package.json

```diff
@@ -47,6 +47,8 @@
     "hast-util-to-string": "^3.0.1",
     "hastscript": "^9.0.1",
     "html-minifier-terser": "^7.2.0",
+    "mdast-util-to-markdown": "^2.1.2",
+    "mdast-util-mdx-jsx": "^3.2.0",
```

Comment:
These are here for testing the output, and will be removed before this is ready for review